flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/CEPITCase.java

```diff
@@ -580,12 +563,17 @@ public String select(Map<String, List<Event>> pattern) {
 			}
 		});
 
-		result.writeAsText(resultPath, FileSystem.WriteMode.OVERWRITE);
+		List<String> resultList = new ArrayList<>();
+
+		DataStreamUtils.collect(result).forEachRemaining(resultList::add);
+
+		List<String> expected = Arrays.asList("1,5,6\n1,2,3\n4,5,6\n1,2,6".split("\n"));
```

Comment:
changes this line to work like other tests, inline `Arrays.asList` into the `assertEquals` call, split the list right away instead of with `split()`.